marlowe-playground-client/src/BlocklyComponent/Types.purs

```diff
@@ -43,16 +44,18 @@ data Query a
   | LoadWorkspace XML a
   | GetBlockRepresentation (Block -> a)
   | SelectWarning Warning a
+  | SetToolbox Toolbox a
 
 data Action
-  = Inject String (Array BlockDefinition)
+  = Inject String (Array BlockDefinition) Toolbox
   | SetData Unit
   | BlocklyEvent BT.BlocklyEvent
   | ResizeWorkspace
   | VisibilityChanged Boolean
 
 data Message
   = CodeChange
+  | BlockSelection (Maybe ({ blockId :: String, blockType :: String }))
```

Comment:
I imagine these parenthesis around the curly brackets are not necessary

Reply:
done